install_config/install/advanced_install.adoc

```diff
@@ -398,6 +398,69 @@ meaning that it is available for placement of new pods. See
 xref:marking-masters-as-unschedulable-nodes[Configuring Schedulability on Masters].
 |===
 
+
+
+[[configuring-cluster-preflight-checks]]
+=== Configuring Cluster Preflight Checks
+
+Preflight checks are a set of diagnostic tasks that run as part of the `OpenShift Health Checks` Ansible role.
+They run prior to an Ansible installation of OpenShift, and ensure that required inventory values are set, and identify potential issues on a host
+that can prevent or interfere with a successful OpenShift installation.
+
+An action plugin for the Health Checks role automatically discovers all checks, and executes only those selected in a play.
+Some checks have tags, which allow for groups of related checks to be run together.
```

Comment:
Suggestion: "Checks may have one or more tags, ..."

In practice, I think all checks so far have at least one tag?